### PR TITLE
ModelApiProcessing first PR

### PR DESCRIPTION
**Pull request by tejkoiejgns**
# Summary: 

## GPT API: 
Firstly, I used the GPT API to summarize the article, and it also paraphrased the article, which made the result more accurate and more inclusive. However, my free trial for $18 has expired. After connecting my billing information to my GPT account, I gained access to the engine, but I was also charged around $4 during these days. Therefore, I decided to use the BART model that I found on Hugging Face.

Here's the pricing information for GPT: https://openai.com/pricing

### For example(GPT):
**Input:** 
https://www.cnn.com/2023/07/11/politics/tommy-tuberville-white-nationalism-cnntv/index.html

**Output:**
Generated phrases:
- Amazon Prime Day is held in July every year to increase sales during slow summer months and to promote early back-to-school and college shopping.
- Amazon(AMZN)held its first Prime Day in 2015 to celebrate its 20th anniversary.
- Prime Day is expected to generate around $12 billion in revenue this year, a 12% increase from last year.
- Amazon is revamping its delivery network this year to create 8 regions in smaller geographic areas designed to ship products over shorter distances.
- Amazon is having its annual Prime Day sale this year, which is expected to draw in more customers than ever.

## BART model: 

1. Use the BART model (https://huggingface.co/facebook/bart-large-cnn) to extract the articles from CNN website
2. Summarize the article
3. Spilt the summarized text into individual sentences 

### For example(BART):
**Input:** 
https://www.cnn.com/2023/07/06/tech/twitter-meta-threads-legal-threat/index.html 

**Output:** 
Summarized Sentences:
1. Twitter is threatening Meta with a lawsuit after the launch of Meta’s new Twitter rival,Threads.
2. The threat is perhaps the clearest sign yet that Twitter views the app as a competitive threat.
3. An attorney representing Twitter sent Meta CEO Mark Zuckerberg a letter that accused the company of trade secret theft.

### CODE REVIEW

File: Phreands_Flask/Backend/Model_API/ModelAPI.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
+from transformers import pipeline
+import requests
+import re
+from bs4 import BeautifulSoup
+
+
+def retrieve_article_text(url):
```
Comment (YLee2121) [inline Phreands_Flask/Backend/Model_API/ModelAPI.py:7]: Refac to Crawler Module in the future

**Comment by YLee2121** [comment]
@tejkoiejgns Could you please add the GPT API charge information in the PR report? I will read through it and tell you my idea.

**Comment by YLee2121** [comment]
and please solve the conflict

**Review by YLee2121 — APPROVED** [review]
approve
please solve the conflict

File: Phreands_Flask/Backend/Model_API/ModelAPI.py (hunk 1 above)
Comment (tejkoiejgns) [reply Phreands_Flask/Backend/Model_API/ModelAPI.py:7]: ok

**Review by alex308248 — APPROVED** [review]
approve

**Pull request merged**